Makefile

```diff
@@ -266,7 +266,7 @@ clean:
 
 assets:
 	@echo ">> writing assets"
-	go generate -x -v ./pkg/promtail/server/ui
+	GOOS=$(shell go env GOHOSTOS) go generate -x -v ./pkg/promtail/server/ui
```

Comment:
I don't think this is necessary? This stage is not compiling any files, it only generates code which will be later compiled in a separate step.  I don't think specifying the operating system at this stage matters?